src/main/java/com/rocana/lucene/codec/v1/RocanaSegmentTermsEnum.java

```diff
@@ -33,19 +33,41 @@
 import org.apache.lucene.util.fst.FST;
 import org.apache.lucene.util.fst.Util;
 
-/** Iterates through terms in this field.  This implementation skips
+/**
+ * Fork of Lucene's {@link org.apache.lucene.codecs.blocktree.SegmentTermsEnum}.
+ *
+ * Why we forked:
+ *   - To use the other forked classes, like {@link RocanaFieldReader}.
+ *
+ * What changed in the fork?
+ *   - Use the other forked classes.
+ *   - Removed trailing whitespace.
+ *   - Changed these javadocs.
+ *
+ * This is one of the forked classes where no logic changed, but to get
+ * the fork to compile we had to fork this class too. That happened with
+ * several classes because they had a hard reference to another class we
+ * forked. Ideally, our forked classes would extend the original Lucene
+ * class and override just the methods we need to change. Unfortunately
+ * in most cases that wasn't an option because many Lucene classes are final.
+ *
+ * See ROCANA-8229
```

Comment:
Same comment here about referring to internal only JIRA tickets in open sourced code.